lightstep-tracer-common

```diff
@@ -1 +1 @@
-Subproject commit a2ac69205156c196ef5dbe7037fda7091580d1e9
+Subproject commit b1bdd09872f4e7d6439f613181997dedb1783458
```

Comment:
checkout-master-and-pull is very different than what I was suggesting.

`git submodule update` should bring the submodule (`lightstep-tracer-common`) back into sync with what the parent repo says about it--it's sort of the analogue of `git checkout` on an ordinary file. 

checkout-master-in-the-submodule-and-pull is like editing a normal file. If you do that you should also run `make proto` to keep the generated file in sync.

(OMG submodules)